### DISCUSSION TITLE
Maybe the solution for battery using

### DISCUSSION THREAD

**Discussion by nevrgithub** [jomjol/AI-on-the-edge-device#2999] [Ideas]
Hello, 

I have seen that battery operation is rather difficult because the power consumption cannot be completely reduced. Would it perhaps be a solution to use a second microcontroller (e.g. a Wemos d1 mini). 

This would e.g. B. control a bistable relay that switches the ESP on for 5 minutes every x hours/days? After that he would of course turn off the power completely. In the meantime, send the Wemos into deep sleep to save energy :) 

I'm looking forward to your opinions on this. 

Best regards Mirko

**Comment by jomjol** [comment]
Idea is okay and already under testing. But the problem is, that there are already no GPIOs to communicate easily with an external board.

**Reply by nevrgithub** [reply]
I would disconnect the complete power input for the Esp32 cam Board with the relay triggered by the the wemos d1  :) there will be no communication between the boards

**Reply by caco3** [reply]
yes, that should workk. It would be like a cold start.
You additionally could implement a MQTT client and subscribe to the messages, so you know when to switch off again.

**Comment by aka13-404** [comment]
> Idea is okay and already under testing. 

Any specifics you could share? I am currently playing around with stuff myself, but esp stuff is not my usual area of expertise.
